Spigot-Server-Patches/0004-MC-Utils.patch

```diff
@@ -2397,8 +2397,7 @@ index b2fb24e9ae19ab6e7039a98fc0c265f801be8a99..44fefe1cdc23667219ce825ba283f84c
      }
  
 +
-+    @Nullable public UUID getUUID(String prefix) { return a(prefix); } // Paper - OBFHELPER
-+    @Nullable
++    public UUID getUUID(String prefix) { return a(prefix); } // Paper - OBFHELPER
      public UUID a(String s) {
```

Comment:
Add documentation saying this _must_ be checked before using, as otherwise it _will_ throw an NPE.

Reply:
ok